cookbooks/aws-parallelcluster-scheduler-plugin/resources/execute_event_handler.rb

```diff
@@ -35,8 +35,10 @@
   cmd.run_command
 
   if cmd.error?
-    raise "Expected Event #{new_resource.event_name} to exit with #{cmd.valid_exit_codes.inspect}," \
-      " but received '#{cmd.exitstatus}', complete log info in #{event_log_out} and error in #{event_log_err}\n #{format_stderr(cmd)}"
+    message = "Expected Event #{new_resource.event_name} to exit with #{cmd.valid_exit_codes.inspect}," \
+      " but received '#{cmd.exitstatus}', complete log info in #{event_log_out} and error in #{event_log_err}"
```

Comment:
I think the events we send to CloudFormation need to be more meaningful for the user creating the cluster. What about something like:
`Failed when running {event} for the configured scheduler plugin. Additional info can be found in /var/log/chef-client.log and {}`